backend/models/Event.js

```diff
@@ -44,6 +53,11 @@ EventSchema.post('save', async function (next) {
   // socket.io emits the event type to the dealership's room so that
   // every running frontend inside the room gets a notification
   const dealership = this.dealership.toString();
+  
+  // simulates the notify() method
+  console.log(this.event_type);
+  console.log("NOW we gonna console log this");
+  console.log(this);
```

Comment:
This can be deleted to not crowd the console

Reply:
I forgot about those. They're deleted in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/64/commits/3d429d429dc99d3842a675389bc0204deb2e4fff)